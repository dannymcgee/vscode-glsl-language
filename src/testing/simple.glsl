#version 440 core

#define PI 3.14;

void main() {
	gl_FragColor = vec4(1,1,1,1);
}
