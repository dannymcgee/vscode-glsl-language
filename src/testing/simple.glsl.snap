// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`simple should work 1`] = `

-> #version 440 core
 | ^ punctuation.definition.directive.glsl  keyword.control.directive.version.glsl  keyword.control.directive.glsl
 |  ^^^^^^^ keyword.control.directive.version.glsl  keyword.control.directive.glsl
 |         ^^^^^^^^^ keyword.control.directive.glsl
-> 
-> #define PI 3.14;
 | ^ punctuation.definition.directive.glsl  keyword.control.directive.define.glsl  meta.preprocessor.macro.glsl
 |  ^^^^^^ keyword.control.directive.define.glsl  meta.preprocessor.macro.glsl
 |        ^ meta.preprocessor.macro.glsl
 |         ^^ entity.name.function.preprocessor.glsl  meta.preprocessor.macro.glsl
 |           ^ meta.preprocessor.macro.glsl
 |            ^ constant.numeric.float.glsl  meta.literal.numeric.float.glsl  meta.preprocessor.macro.glsl
 |             ^ punctuation.separator.numeric.decimal.glsl  meta.literal.numeric.float.glsl  meta.preprocessor.macro.glsl
 |              ^^ constant.numeric.float.glsl  meta.literal.numeric.float.glsl  meta.preprocessor.macro.glsl
 |                ^ punctuation.terminator.glsl  meta.preprocessor.macro.glsl
-> 
-> void main() {
 | ^^^^ storage.type.primitive.void.glsl
 |      ^^^^ entity.name.function.glsl
 |          ^ punctuation.brace.round.glsl
 |           ^ punctuation.brace.round.glsl
 |             ^ punctuation.brace.curly.glsl
->     gl_FragColor = vec4(1,1,1,1);
 |     ^^^^^^^^^^^^ variable.language.reserved.glsl
 |                  ^ keyword.operator.assignment.glsl
 |                    ^^^^ entity.name.type.other.glsl
 |                        ^ punctuation.brace.round.glsl
 |                         ^ constant.numeric.integer.decimal.glsl  meta.literal.numeric.decimal.glsl
 |                          ^ punctuation.separator.comma.glsl
 |                           ^ constant.numeric.integer.decimal.glsl  meta.literal.numeric.decimal.glsl
 |                            ^ punctuation.separator.comma.glsl
 |                             ^ constant.numeric.integer.decimal.glsl  meta.literal.numeric.decimal.glsl
 |                              ^ punctuation.separator.comma.glsl
 |                               ^ constant.numeric.integer.decimal.glsl  meta.literal.numeric.decimal.glsl
 |                                ^ punctuation.brace.round.glsl
 |                                 ^ punctuation.terminator.glsl
-> }
 | ^ punctuation.brace.curly.glsl
-> 
`;
